src/main/rx/rx.c

```diff
@@ -877,6 +878,28 @@ void setRssiDbmDirect(int16_t newRssiDbm, rssiSource_e source)
     rssiDbm = newRssiDbm;
 }
 
+static int16_t updateRsnrSamples(int16_t value)
+{
+    static int16_t samplessnr[RSSI_SAMPLE_COUNT_DBM];
+    static uint8_t samplesnrIndex = 0;
+    static int sumsnr = 0;
+
+    sumsnr += value - samplessnr[samplesnrIndex];
+    samplessnr[samplesnrIndex] = value;
+    samplesnrIndex = (samplesnrIndex + 1) % RSSI_SAMPLE_COUNT_DBM;
+    return sumsnr / RSSI_SAMPLE_COUNT_DBM;
+}
+
+void setRsnr(int16_t rsnrValue)
+{
+    rsnr = updateRsnrSamples(rsnrValue);
+}
+
+void setRsnrDbmDirect(int16_t newRsnr)
```

Comment:
should be `setRsnrDirect`

Reply:
which means noone is calling setRsnrDirect (correctly declared in rx.h), otherwise the compiler would have complained :-) anyway, fixing it, thanks!